packages/flutter/lib/src/material/drawer_header.dart

```diff
@@ -2,16 +2,21 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 
 import 'debug.dart';
 import 'theme.dart';
 
-const double _kDrawerHeaderHeight = 140.0;
+const double _kDrawerHeaderHeight = 160.0;
```

Comment:
This size is not documented, so I'm not completely sure if it's the right value or not. Any ideas here?